nemo_skills/evaluation/evaluator/mcq.py

```diff
@@ -45,6 +45,10 @@ def extract_letter(text):
             data = [json.loads(line) for line in fin]
         with open(file, "wt", encoding="utf-8") as fout:
             for sample in tqdm(data):
-                sample["predicted_answer"] = extract_letter(sample["generation"])
+                extract_from_boxed = sample.get("extract_from_boxed", "True") == "True"
```

Comment:
why don't we use True / False as actual values, not strings? json / omegaconf all support this and feels a bit less error-prone that way. So this applies both here and inside prepare.py